### DISCUSSION TITLE
"avoid _negative_ts" error

### DISCUSSION THREAD

**Discussion by xcodelight** [mifi/lossless-cut#2007] [Support] [answered]
If i set it to auto while merging , i get this error 
![image](https://github.com/mifi/lossless-cut/assets/120263284/0c6cf896-ac35-4455-b5e4-2ca66f8dc94c)

If i set it to value other than auto while not merging, i get this error
![image](https://github.com/mifi/lossless-cut/assets/120263284/acc8c452-9926-4a57-86b9-01d2b18baac2)

**Comment by mifi** [comment, marked as answer]
It is not an error. It's a warning
